mlbox/preprocessing/drift/rde_cv.py

```diff
@@ -0,0 +1,345 @@
+# # coding: utf-8
```

Comment:
This file is completely commented, it should be remove and included in the next version

Reply:
ok it will be deleted on master but will stay on dev branch...